### PR TITLE
Update registry.json with GLOW

### PR DESCRIPTION
**Pull request by xbglowx**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -4017,18 +4017,31 @@
...
+   "telegram": "missawCBFR",
+   "operatorName": "Wassim",
+   "github": "missawZero"
+  },
+  {
+   "ticker": "GLOW",
+   "poolId": "5dc1f763f14baaabc8d14aa9c0717e8d4662e4d16b2e10095d7ec78b",
+   "social": {
+     "website": "https://cardano.xbglowx.com",
+     "twitter": "CardanoGlow",
```
Comment (Bez625) [inline registry.json:4038]: There is no activity on this Twitter account and the website doesnt seem to load anything for me at present. We ask that all members have some form of social media presence. Please can you add some activity and I will re-review? Thanks

**Review by Bez625 — CHANGES_REQUESTED** [review]

File: registry.json (hunk 1 above)
Comment (xbglowx) [reply registry.json:4038]: Thanks for the review.

Still working on getting everything configured, hence the lack of Twitter activity and https://cardano.xbglowx.com not loading. 

I will ping once I have fixed your concerns.

Comment (xbglowx) [reply registry.json:4038]: Can you please have another look?

Comment (xbglowx) [reply registry.json:4052]: Sorry to ping again. Ready for my close up.

**Review by Bez625 — APPROVED** [review]

**Pull request merged**